halpybot/packages/command/commandhandler.py

```diff
@@ -232,7 +232,7 @@ async def invoke_command(
         if isinstance(cmd, CommandGroup):  # Command group
             subgroup = CommandGroup.get_group(name=cmd.name)
             # If no subcommand is provided, send a provisional help response
-            if len(arguments) < 1:
+            if not arguments:
```

Comment:
sanity check: the nature of this check changed. confirm this is intended!

Reply:
Confirmed.